specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/services.json

```diff
@@ -0,0 +1,1832 @@
+                }
+              ]
+            },
+            "containerResourceRequirements": {
+              "description": "The container resource requirements.",
+              "$ref": "machineLearningServices.json#/definitions/ContainerResourceRequirements"
+            },
+            "maxConcurrentRequestsPerContainer": {
+              "format": "int32",
+              "description": "The maximum number of concurrent requests per container.",
+              "type": "integer",
+              "example": 100
+            },
+            "maxQueueWaitMs": {
+              "format": "int32",
+              "description": "Maximum time a request will wait in the queue (in milliseconds). After this time, the service will return 503 (Service Unavailable)",
+              "type": "integer",
+              "example": 250
+            },
+            "namespace": {
+              "description": "Kubernetes namespace for the service.",
+              "type": "string",
+              "example": "default"
+            },
+            "scoringTimeoutMs": {
+              "format": "int32",
+              "description": "The scoring timeout in milliseconds.",
+              "type": "integer",
+              "example": 100
+            },
+            "authEnabled": {
+              "description": "Whether or not authentication is enabled.",
+              "type": "boolean"
+            },
+            "livenessProbeRequirements": {
+              "description": "The liveness probe requirements.",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/LivenessProbeRequirements"
+                }
+              ]
+            },
+            "aadAuthEnabled": {
+              "description": "Whether or not AAD authentication is enabled.",
+              "type": "boolean"
+            }
+          }
+        }
+      ],
+      "x-ms-discriminator-value": "AKS"
+    },
+    "CreateEndpointVariantRequest": {
+      "description": "The Variant properties.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CreateServiceRequest"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "isDefault": {
+              "description": "Is this the default variant.",
+              "type": "boolean"
+            },
+            "trafficPercentile": {
+              "format": "float",
+              "description": "The amount of traffic variant receives.",
+              "type": "number",
+              "example": 100
+            },
+            "type": {
+              "description": "The type of the variant.",
+              "enum": [
+                "Control",
+                "Treatment"
+              ],
+              "type": "string",
+              "example": "Control",
+              "x-ms-enum": {
+                "name": "VariantType",
+                "modelAsString": true
+              }
+            }
+          }
```

Comment:
Same as above

Reply:
we are deprecating services.json so I will leave it as it is